app/src/main/java/org/p2p/wallet/infrastructure/network/NetworkModule.kt

```diff
@@ -92,6 +93,15 @@ object NetworkModule : InjectionModule {
                 interceptor = RpcInterceptor(get(), get())
             )
         }
+        single<RpcApi> {
```

Comment:
moved RpcApi to our module because there was not enough logging for `update_metadata` requests